storybook-utilities/storybook-theming/scss/_spark-angular-manifest.scss

```diff
@@ -1,2 +1 @@
-@import '../../../html/spark.scss';
-@import '../../../html/spark-angular.scss';
+@import '../../../styles/spark.scss';
```

Comment:
I moved spark-angular sass into spark.scss so we dont need to individually import angular sass anymore.